transports/msmq/distributor/configuration.md

```diff
@@ -76,3 +76,87 @@ If self-hosting the endpoint here is the code required to enlist the endpoint wi
+
+#### Sender side distribution
+
+Using [sender side distribution](h/transports/msmq/sender-side-distribution) instead of using the distributor might be a better alternative if you are able to use NServiceBus 6 and is much more suitable for scenarios where IO is the bottleneck. 
+
+#### Do not run all distributors on a single box
+
+Distributor endpoints are often deployed on *the distributor machine*. This is one machine that hosts all distributor endpoints. This becomes the bottleneck under load as all messages flow throught this machine. An alternative is to have multiple distributor machines on physically different networks interface cards.
+
+Alternative, when first hosting all endpoints on a single machine do not use the distributor to now run *all* endpoints on multiple workers. First balance your current endpoints by moving endpoints. Eventually you will have only a single endpoint running on a machine. If this machine now becomes your bottleneck then you might first want to upscale this first and if upscaling becomes too costly then make use of the distributor. 
+
+#### Upscaling
+
+The distributor is very chatty. Network latency and bandwidth will lower the overal throughput when the CPU isn't your current bottleneck, using the distributor might not be the ideal solution as using the distributor will result in roughly 4 times the number of send messages thus has a severy impact on your network resources.
+
+
+### Each worker is getting events directly, bypassing the distributor 
+
+The worker is very likely to not be running as a worker, but as a regular endpoint. In this configuration the worker will subscribe using its own address and not using the distributor.
+
+Make sure that:
+
+- The assembly `NServiceBus.Distributor.MSMQ.dll` is in the endpoint folder.
+- When using the [NServiceBus.Host](/nservicebus/hosting/nservicebus-host/), the `NServiceBus.MSMQWorker` profile is set on the commandline and that the worker was installed with this argument on the commandline.
+- When using [custom assembly scanning](/nservicebus/hosting/assembly-scanning/), the in/exclusions are not excluding the `NServiceBus.Distributor.MSMQ.dll` assembly.
+  - The log should have the following feature, and is easily found when searching on **Distributor**:
+```  
+Name: Distributor
+Version: 5.0.5
+Enabled by Default: No
+Status: Enabled
+Dependencies: None
+Startup Tasks: None
+```
+
+### The distributor queue is building up while it is running
+
+The distributor is probably unaware of any workers. When a worker starts it enlists its capacity at the distributor and this event is written to the log.
```

Comment:
> it enlists its capacity at the distributor

simpler would be "it signals that it's available to the distributor" or similar

Reply:
Add log entry text